.github/workflows/lcm_dreamshaper_cpp.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         run: |
           conda activate openvino_lcm_cpp
           conda update -c conda-forge --all
-          conda install -c conda-forge -c conda-forge/label/openvino_dev openvino==2024.2.0.dev20240513 c-compiler cxx-compiler git make cmake
+          conda install -c conda-forge -c conda-forge/label/openvino_dev openvino==2024.3.0.dev20240614 c-compiler cxx-compiler git make cmake
```

Comment:
let's wait for https://github.com/conda-forge/openvino-feedstock/runs/26751639883 to complete

Reply:
Passed